### PR TITLE
Add home panel

### PR DESCRIPTION
**Pull request by EhsanMrh**

### CODE REVIEW

File: src/components/Pages/Panel/PanelComponents/PanelNavbar.vue [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            searchValue: ""
+        }
+    },
+    methods: {
+        ...mapActions([
+            "logout"
+        ]),
+        onLogout() {
+            return this.logout()
+                    .then(() => this.$router.push("/"))
```
Comment (pmzi) [inline src/components/Pages/Panel/PanelComponents/PanelNavbar.vue]:
```
this.$router.replace('/')
```

File: src/components/Pages/Panel/PanelComponents/PanelNavbar.vue [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        },
+        onSearch() {
+            console.log(this.searchValue);
+        }
+    }
+}
+</script>
+
+<style lang="scss"> 
+.ant-layout-header {
```
Comment (pmzi) [inline src/components/Pages/Panel/PanelComponents/PanelNavbar.vue]: BEM....

File: src/router/index.js [hunk 3]
```diff
@@ -1,10 +1,29 @@
...
 import Config from "./config";
 
 Vue.use(Router);
 
-// add function for initializing
+const router = new Router(Config);
 
-export default new Router(Config);
+router.beforeEach((to, from, next) => {
+  const accessToken = store.getters.isAuth;
```
Comment (pmzi) [inline src/router/index.js]: Vue.prototype.$store

File: src/store/modules/login.js [hunk 4]
```diff
@@ -1,60 +1,61 @@
...
+      commit("UPDATE_ACCESS_TOKEN", localStorage.getItem("accessToken"));
+    },
+    logout({ commit }) {
+      localStorage.removeItem("accessToken");
+      commit("LOGOUT");
+    }
+  },
+  getters: {
+    isAuth: state => {
+      return !!state.accessToken;
```
Comment (pmzi) [inline src/store/modules/login.js]: Boolean. Is it needed to be a boolean at all??

File: src/router/index.js (hunk 3 above)
Comment (EhsanMrh) [reply src/router/index.js]: it doesn't work

**Pull request merged**